src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -175,31 +198,33 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
   }, [itemToShare, roles]);
 
   const loadShareInfo = async () => {
+    if (!itemToShare?.item) return;
+
+    const isItemNotSharedYet = !isAdvanchedShareItem(itemToShare?.item) && !itemToShare.item.sharings?.length;
+    if (isItemNotSharedYet) return;
+
```

Comment:
Avoids unnecesary calls to the endpoint that returns the sharing data and sharing type.

Types here are kind of problematic here, they don't align with the current backend response and are heavily nested. So I used the typeguard `isAdvanchedShareItem` to be able to access to "item.sharings". 

Current item fetched from backend:

`{
    "id": 2013,
     ..... , 
    "shares": [],
    "sharings": []
}`